internal/command/command.go

```diff
@@ -424,14 +431,40 @@ func UsageMetrics(command *cobra.Command, wg *sync.WaitGroup) {
 
 // GlobalFlags contains all global flags definitions.
 type GlobalFlags struct {
-	Filter           string
-	Format           string
-	Save             string
-	Host             string
-	HostNetworkKey   string
-	Log              string
-	Network          string
-	Yes              bool
-	ConfigPaths      []string
-	SkipVersionCheck bool
+	Filter                     string
+	Format                     string
+	Save                       string
+	Host                       string
+	HostNetworkKey             string
+	Log                        string
+	Network                    string
+	Yes                        bool
+	ConfigPaths                []string
+	SkipVersionCheck           bool
+	SkipContractMigrationCheck bool
+}
+
+func checkContractMigrations(state *flowkit.State, logger output.Logger, flow flowkit.Services) {
+	contractStatuses, err := validator.NewValidator(github.NewClient(nil).Repositories, flow.Network(), state, logger).GetContractStatuses()
+	if err != nil {
+		logger.Error(fmt.Sprintf("failed to validate contracts: %s", err))
+		return
+	}
+
+	for _, contract := range contractStatuses {
+		if contract.IsFailure() {
+			logger.Error(fmt.Sprintf(
+				"Contract %s has failed the last emulated migration\n"+
+					" - Account: %s\n"+
+					" - Contract: %s\n"+
+					" - Error: %s\n",
+				contract.ContractName,
+				contract.AccountAddress,
+				contract.ContractName,
+				contract.Error,
+			))
```

Comment:
Can we print this to stderr (the default logger here is an `StdoutLogger`).  This is diagnostic information intended for the user and not necessarily comand output.  Using stdout could break external tooling that relies on the output from the Flow CLI (e.g. Language Server, VSCode, Flow JS Testing).